presto-hive/src/test/java/com/facebook/presto/hive/TestHiveFileFormats.java

```diff
@@ -333,43 +321,6 @@ public void testAvro(int rowCount)
                 .collect(toList());
     }
 
-    @Test(dataProvider = "rowCount")
```

Comment:
What's the reason for removing these tests?

Reply:
these tests are for old Parquet reader only